app/app.go

```diff
@@ -635,8 +635,9 @@ func NewApp(
 	app.communityKeeper = communitykeeper.NewKeeper(
 		app.accountKeeper,
 		app.bankKeeper,
+		&cdpKeeper,
 		app.distrKeeper,
-		hardKeeper,
+		&hardKeeper,
```

Comment:
creating a public record of the fact that because this was originally passed as a copy, any positions it created never had claims instantiated (because the hooks do not get registered to the copy).
this may have consequences for the open lend position held by the community module.

Reply:
confirmed with @rhuairahrighairidh any fixes needed for this would be in a follow up pr. commencing merge!